localstack/testing/pytest/fixtures.py

```diff
@@ -1131,6 +1131,154 @@ def _check_logs(func_name: str, expected_lines: List[str] = None):
     return _check_logs
 
 
+@pytest.fixture
+def wait_for_user():
+    def _wait(keys):
+        sts_client = create_client_with_keys(service="sts", keys=keys)
+
+        def is_user_ready():
+            try:
+                sts_client.get_caller_identity()
+                return True
+            except ClientError as e:
+                if e.response["Error"]["Code"] == "InvalidClientTokenId":
+                    return False
+                return True
+
+        # wait until the given user is ready, takes AWS IAM a while...
+        poll_condition(is_user_ready, interval=5, timeout=20)
+
+    return _wait
+
+
+@pytest.fixture
+def create_policy(iam_client):
+    policy_arns = []
+
+    def _create_policy(*args, **kwargs):
+        if "PolicyName" not in kwargs:
+            kwargs["PolicyName"] = f"policy-{short_uid()}"
+        response = iam_client.create_policy(*args, **kwargs)
+        policy_arn = response["Policy"]["Arn"]
+        policy_arns.append(policy_arn)
+        return response
+
+    yield _create_policy
+
+    for policy_arn in policy_arns:
+        try:
+            iam_client.delete_policy(PolicyArn=policy_arn)
+        except ClientError:
```

Comment:
I think it's fine to open this except clause up a bit more since the primary purpose of catching it here is to allow the deletion of the other policies afterwards